src/lxc/freezer.c

```diff
@@ -65,19 +65,21 @@ static int do_freeze_thaw(bool freeze, struct lxc_conf *conf, const char *name,
 	ret = cgroup_ops->set(cgroup_ops, "freezer.state", state, name, lxcpath);
 	if (ret < 0) {
 		cgroup_exit(cgroup_ops);
-		ERROR("Failed to %s %s", (new_state == FROZEN ? "freeze" : "unfreeze"), name);
+		ERROR("Failed to %s %s",
+		      (new_state == FROZEN ? "freeze" : "unfreeze"), name);
```

Comment:
TBH if you wanted to actually improve this, it would be better to use a temporary variable with a meaningful name, assigned in the previous line.